src/calculateRentalCost.js

```diff
@@ -27,8 +27,35 @@
  * @return {number}
  */
 
+const RENT_COST = 40;
+const FIRST_OFF = 20;
+const SECOND_OFF = 50;
+const LONG_TERM_3 = 3;
+const LONG_TERM_6 = 6;
+const LONG_TERM_7 = 7;
+
+
+
 function calculateRentalCost(days) {
-  // write code here
+  let finalCost;
+
+  if (days < LONG_TERM_3) {
+    finalCost = days * RENT_COST;
+
+    return finalCost;
+  }
+
+  if (days >= LONG_TERM_3 && days <= LONG_TERM_6) {
```

Comment:
if you check biggest value first of all it will allow you to get rid of `&&` here